src/pages/Game.tsx

```diff
@@ -171,6 +177,18 @@ const GamePage = ({ ws, currentUser }: GameProps): JSX.Element => {
     const mjPlayer = mjGameState.getMjPlayer();
 
     mjPlayer.addTileToHand(tile);
+
+    // Send SELF_PLAY_TILE if tile is a Bonus Tile
+    if (isBonusTile(data.tile)) {
+      mjPlayer.getHand().setCannotPlayTile();
+      const wsPayload = {
```

Comment:
can force it to be the same as the `SelfPlayTileJSON` interface:
```ts
const wsPayload: SelfPlayTileJSON = {
}
```

Reply:
Payload interfaces are for messages incoming to the client. Message validator implicitly? explicitly? checks the keys without using an interface